roles/redhat_satellite6_lb/vars/main.yml

```diff
@@ -0,0 +1,12 @@
+---
```

Comment:
@Smithe2413 these should all be in a defaults/main so someone can override them. in vars/main it is harder to override. https://docs.ansible.com/ansible/latest/user_guide/playbooks_variables.html#variable-precedence-where-should-i-put-a-variable

Reply:
Sounds good. I'll update it. Should all the variables be moved or should some remain in `/vars/main.yml`?